buildkite/src/Command/DeployNightly.dhall

```diff
@@ -0,0 +1,23 @@
+let Prelude = ../External/Prelude.dhall
+
+let Command = ./Base.dhall
+let Size = ./Size.dhall
+
+let Cmd = ../Lib/Cmds.dhall in
+
+{ step = \(dependsOn : List Command.TaggedKey.Type) ->
+    Command.build
+      Command.Config::{
+        commands = [
+          Cmd.runInDocker
+            Cmd.Docker::{
+              image = (../Constants/ContainerImages.dhall).codaToolchain
+            }
+            "cd coda-automation/terraform/testnets/nightly && terraform init; terraform destroy -auto-approve; terraform apply -auto-approve"
```

Comment:
can you stick this in a `buildkite/scripts/` script?

Reply:
:+1: